site/content/docs/manifest/worker-service.en.md

```diff
@@ -95,13 +95,13 @@ If specified, creates a dead letter queue and a redrive policy which routes mess
 <span class="parent-field">subscribe.</span><a id="subscribe-topics" href="#subscribe-topics" class="field">`topics`</a> <span class="type">Array of `topic`s</span>  
 Contains information about which SNS topics the worker service should subscribe to.
 
-<span class="parent-field">topic.</span><a id="topic-name" href="#topic-name" class="field">`name`</a> <span class="type">String</span>  
+<span class="parent-field">subscribe.topics.</span><a id="topic-name" href="#topic-name" class="field">`name`</a> <span class="type">String</span>  
```

Comment:
This is inconsistent with:

https://github.com/aws/copilot-cli/blob/87eec019eb3d9c1ecc78fcb20d2bb47759f4b406/site/content/docs/include/publish.en.md?plain=1#L26

I'm not sure which one is "correct", but we should probably stay consistent.